modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/util/SamlUtil.java

```diff
@@ -447,4 +455,7 @@ public static SingleSignOnService resolveSingleSignOnService(
 		return null;
 	}
 
+	private static final String _EXPLICITLY_MAPPED_ATTRIBUTE_NAMES =
+		"membership:userGroups";
+
```

Comment:
This circumvents the SPI. Because we have decided to introduce a new concept of some userFieldExpressions not allowing implicit mappings, we should enable customers to utilize that in their own `UserFieldExpressionHandlers` also.